docs/tutorials/federation/set-up-coredns-provider-federation.md

```diff
@@ -0,0 +1,126 @@
+---
+assignees:
+- shashidharatd
+title: Setting up CoreDNS as DNS provider for Cluster Federation
+---
+
```

Comment:
If you could please use the [Tutorial Template](https://kubernetes.io/docs/home/contribute/page-templates/#tutorial_template), that would really help us out. Thanks!